src/task-implementation.rst

```diff
@@ -50,22 +50,22 @@ independent testing, re-use, and shell mode editing.
 Task Messages
 -------------
 
-Tasks messages can be sent back to the suite server program to report completed
-outputs and arbitrary messages of different severity levels.
-
-Some types of message - in addition to events like task failure -  can
-optionally trigger execution of event handlers in the suite server program
-(see :ref:`EventHandling`).
+Task jobs automatically send status messages back to the server program to
```

Comment:
Clarify that this doesn't happen if you set task communication method = poll ?

Reply:
Done.